src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -168,7 +168,7 @@ internal ImmutableList<string> Deployments(RoleName node)
             return deployments == null ? _allDeploy : deployments.AddRange(_allDeploy);
         }
 
-        internal ImmutableList<RoleName> Roles
+        public ImmutableList<RoleName> Roles
```

Comment:
Made this property public so the data is just available to anyone who needs it - preferred this over friend assembly magic.